src/apps/games/tasks/refresh_materialized_game_views.py

```diff
@@ -0,0 +1,13 @@
+from src import celery_app
+
+from src.apps.games.models import GameCountByNetwork, GameCountByUser
+
+
+@celery_app.task()
+def refresh_materialized_game_views():
+    """
+    Refresh all the materialized views for games app
+    :return:
+    """
+    GameCountByNetwork.refresh(concurrently=True)
```

Comment:
nice

Reply:
Is there a way to automatically set up the celery task with a default period upon migrate or do I just do that manually in admin panel?